assets/snippets/docinfo/snippet.docinfo.php

```diff
@@ -18,10 +19,22 @@
 *       [[DocInfo? &docid=`15` &field=`pagetitle`]]
 *       [[DocInfo? &docid=`10` &field=`tvname`]]
 *       [[DocInfo? &docid=`3` &field=`tvname` &render=`1`]]
+*       [[DocInfo? &docid=`3` &templid=`2` &field=`tvname`]]
 */
 if(!defined('MODX_BASE_PATH')){die('What are you doing? Get out of here!');}
 $default_field = array('type','contentType','pagetitle','longtitle','description','alias','link_attributes','published','pub_date','unpub_date','parent','isfolder','introtext','content','richtext','template','menuindex','searchable','cacheable','createdon','createdby','editedon','editedby','deleted','deletedon','deletedby','publishedon','publishedby','menutitle','donthit','privateweb','privatemgr','content_dispo','hidemenu','alias_visible');
 $docid = (isset($docid) && (int)$docid>0) ? (int)$docid : $modx->documentIdentifier;
+if(isset($templid) && (int)$templid>0) {
+	$doc['parent'] = $docid;
+	$limit = 20;
```

Comment:
Maybe ```$limit``` must also changed via parameters?

Reply:
It's a good idea! &limit parameter was added.
And default limit was decreased to 10, because I saw the same limit in the getChildIds method:
function getChildIds($id, $depth= 10, $children= array ()) {
http://modx-gu.ru/api/getchildids